scripts/util/util.py

```diff
@@ -13,26 +13,11 @@
 import subprocess
 
 
-class LyBuildError(Exception):
```

Comment:
You also changed the permissions for this file and the `scripts/util/glob_to_regex.py` to remove the execute bit.

Did you intend to do that?

Reply:
did not intend, but these scripts should be executed from `python.cmd` anyway, so better if they dont have the execute bit. 
they are all loaded from other py scripts, so they dont need it, right?

Reply:
Yes the execute bit is not needed if the script is not invoked directly by the loader.
It just means that `./util.py` will not work, but `python.cmd util.py` will always work